Databricks/Dockerfile

```diff
@@ -109,17 +109,27 @@ RUN chmod 755 /opt/spark-rapids/init.sh
 
 WORKDIR /databricks
 
+FROM ubuntu:18.04 as download-modpython
+
+RUN cd /tmp && \
+  apt-get -y update && \
+  apt-get -y download libganglia1 && \
+  ls -al
```

Comment:
why the ls here?

Reply:
Added the comment. TL;DR reason is because of a known packaging bug in Ubuntu 20.04 and higher (see https://bugs.launchpad.net/ubuntu/+source/ganglia/+bug/1894045) the ganglia python support is broken, so we have to insert modpython.so from Ubuntu 18.04 to make this work.